client/plots/matrix/matrix.legend.js

```diff
@@ -143,6 +143,54 @@ export function getLegendData(legendGroups, refs, self) {
 		const keys = Object.keys(legend.values).sort()
 		const hasScale = Object.values(legend.values).find(v => v.scale)
 		if (hasScale) {
+			if (legend === legendGroups?.CNV) {
+				/** Combine gain and loss here to avoid rendering issues */
+				const legendValues = Object.values(legend.values)
+				const findLegend = type =>
```

Comment:
should use `v.key === 'CNV_loss'` (or use constant value from `common.js`) instead of comparing against `v.label` which is arbitrary.

Reply:
removed this code in my clean up, so no need to adress